### ISSUE TITLE
"slurp" is not exported by the Mojo::Util module

### ISSUE THREAD

**Issue by eserte** [niczero/mojar-message#2]
Another one:
```
"slurp" is not exported by the Mojo::Util module
Can't continue after import errors at test/version.t line 6.
BEGIN failed--compilation aborted at test/version.t line 6.
test/version.t ............ 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
```

**Comment by eserte** [comment]
Still fails with 1.102.

**Comment by eserte** [comment]
It does not fail if Mojar 2.101 is installed, so probably just the minimum prereq version has to be set:
```
****************************************************************
Regression 'mod:Mojar'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	     -0.0000	      0.0000	  -8.18
[1='eq_2.101'] 	      1.0000	      0.0000	50495178523819704.00

R^2= 1.000, N= 49, K= 2
```

**Comment by niczero** [comment]
Thank you for contributing @eserte.

This was fixed in v1.102
https://travis-ci.org/niczero/mojar-message/builds/238050235
